libraries/typescript/packages/evals/README.md

```
@@ -0,0 +1,459 @@
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/mcp-use/mcp-use-ts/main/packages/mcp-use/static/logo_white.svg">
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/mcp-use/mcp-use-ts/main/packages/mcp-use/static/logo_black.svg">
+    <img alt="mcp use logo" src="https://raw.githubusercontent.com/mcp-use/mcp-use-ts/main/packages/mcp-use/static/logo_white.svg" width="80%" style="margin: 20px auto;">
+  </picture>
+</div>
+
+<h1 align="center">@mcp-use/evals</h1>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@mcp-use/evals" alt="NPM Downloads">
+        <img src="https://img.shields.io/npm/dw/@mcp-use/evals.svg"/></a>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)